templates/panels-development.ini.erb

```diff
@@ -0,0 +1,23 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+<% if @hide_schedule %>
+hide_schedule = <%= @hide_schedule %>
+<% end -%>
+
+<% if @expected_response %>
+expected_response = "<%= @expected_response %>"
+<% end -%>
+
+<% if @event_location %>
+event_location = "<%= @event_location %>"
+<% end -%>
+
+<% if @panel_rooms %>
+panel_rooms = <%= @panel_rooms %>
+<% end -%>
+
+<% if @panel_app_deadline %>
+[dates]
+panel_app_deadline = "<%= @panel_app_deadline %>"
+<% end -%>
```

Comment:
same deal, there's a lot more to add here to get this working, but it's a good starting point